### DISCUSSION TITLE
Streamlit Component

### DISCUSSION THREAD

**Discussion by 1Cor134** [bcakmakoglu/vue-flow#421] [Q&A] [answered]
I'm trying to make a Streamlit component out of vue-flow.  I'm a novice at Vue and Streamlit and am running into problems.

Other Streamlit components for Vue 3 projects like yours end up doing something like this:

import something from "some-vue3"

const app = createApp(App)
    .use(something)

Where "something" is a Vue Plugin Install Function.

I was unable to find a PluginInstallFunction for vue-flow, something like this:
          declare const install: Exclude<Plugin["install"], undefined>

I don't suppose you've seen or created a Streamlit component for vue-flow, or would be able to help me resolve this?

thanks!

**Comment by bcakmakoglu** [comment, marked as answer]
There is no plugin to install with Vue Flow.
Plugins are *not* components. They can "install" components to your global app instance but they are themselves not components.

You can just import Vue Flow wherever you need to use it. Again: no plugin required to use Vue Flow.
